packages/theme/src/cli/utilities/theme-fs.ts

```diff
@@ -65,10 +65,18 @@ export async function mountThemeFileSystem(root: string): Promise<ThemeFileSyste
   return {
     root,
     files,
+    delete: async (assetKey: string) => {
+      await removeThemeFile(root, assetKey)
+      files.delete(assetKey)
+    },
+    write: async (asset: ThemeAsset) => {
+      await writeThemeFile(root, asset)
+      files.set(asset.key, asset)
+    },
   }
 }
 
-export async function writeThemeFile(root: string, {key, attachment, value}: ThemeAsset) {
+async function writeThemeFile(root: string, {key, attachment, value}: ThemeAsset) {
```

Comment:
This is being changed to `private` to avoid bugs where we remove the file but forget to update the `ThemeFileSystem`

Previously, users would have to explicitly call this, which is easy to forget
```js
await writeThemeFile(root, asset)
themeFileSystem.files.set(asset.key, asset)
```